it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationControllerISpec.scala

```diff
@@ -185,21 +190,6 @@ trait BusinessIdentificationControllerISpec extends BaseISpec with SessionDataMi
           sessionStoreService,
           userIsAuthenticated(subscribingCleanAgentWithoutEnrolments))
       }
-      "no businessType parameter supplied" should {
```

Comment:
Still want a test to cover this case when the parameter is not given. Should still redirect to `showBusinessTypeForm` page in order to support those UI journeys that are still linking directly to the old URL (from a time before we created the show business type page).

Reply:
having discussed this with you, and regarding the comment below(same deal) this will be assured using the binder